infra/modules/service/dns.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_route53_record" "app" {
+  # Don't create DNS record for temporary environments (e.g. ones spun up by CI/)
+  count = !var.is_temporary && var.domain_name != null && var.hosted_zone_id != null ? 1 : 0
+
+  name    = var.domain_name
```

Comment:
I wasn't sure if AWS supports having a full domain name here (e.g. `www.example.com` as opposed to `www`). It looks like it from the terraform examples, did you confirm it works?

Reply:
Yeah I confirmed it, see the screenshot on the platform-test PR based on my testing there, pasted here as as well:
![image](https://github.com/navapbc/template-infra/assets/447859/173a4f45-59d5-4eaa-b6f2-63c3eed16921)

I also did this same approach on another internal project

Reply:
Also, I basically copied your old PR so thanks for that work!